src/firefly/js/ui/tap/TapUtil.js

```diff
@@ -287,13 +287,20 @@ export const getAsEntryForTableName= (tableName) => tableName?.[0] ?? 'x';
 function mergeAdditionalServices(tapServices, additional) {
     if (!hasElements(additional)) return tapServices;
 
-    const modifiedOriginal= tapServices.map( (t) => {
-        const match= additional.find( (a) => a.label===t.label);
-        return match ? {...t,...match} : t;
-    });
-    const trulyAdditional= additional.filter( (a) => !tapServices.find( (t) => t.label===a.label) );
-
-    return [...trulyAdditional,...modifiedOriginal];
+    const mergeAdditional= additional.map( (a) => {
+        if (a.hide) return false;
```

Comment:
I'm not sure how to use "hide" in the JSON, and it looks like I'll want to in order to provide the cleanest user experience.  Ca you provide an example?